examples/validationCycle2/src/tga2_likelihood.C

```diff
@@ -58,6 +58,8 @@ likelihoodRoutine_Data::likelihoodRoutine_Data(
     FILE *inp;
     inp = fopen(inpName1,"r");
 
+    queso_require_msg(inp, "Unable to open" << inpName1);
```

Comment:
Can you put a space between `n` and `"` so the name of the input file prints as a separate word?